aws/terminator/data_services.py

```diff
@@ -158,3 +158,24 @@ def created_time(self):
 
     def terminate(self):
         self.client.delete_cluster(ClusterIdentifier=self.id, SkipFinalClusterSnapshot=True)
+
+
+class RdsOptionGroup(DbTerminator):
+    @staticmethod
+    def create(credentials):
+        return Terminator._create(credentials, RdsOptionGroup, 'rds', lambda client: client.describe_option_groups()['OptionGroupsList'])
+
+    @property
+    def id(self):
+        return self.instance['OptionGroupArn']
+
+    @property
+    def name(self):
+        return self.instance['OptionGroupName']
+
+    @property
+    def ignore(self):
+        return not self.name.startswith('default')
```

Comment:
```suggestion
        return self.name.startswith('default')
```
This one is a little confusing because it's kind of a double negative (I had to experiment with it a bit myself).  This function returns a list of things _to be ignored_, not a list of things that excludes what to ignore.  So we want to return only those things which start with `default`.

Reply:
Ok, done. Thank you.